src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -130,6 +132,7 @@ protected override Task InitializeAsync(CancellationToken cancellationToken, IPr
         {
             AddService(typeof(IGitHubServiceProvider), CreateService, true);
             AddService(typeof(IUsageTracker), CreateService, true);
+            AddService(typeof(ILoginManager), CreateService, true);
```

Comment:
Actually, I think it could. I think it was originally a service as it had an async initialization, but that's not there any more.